### PR TITLE
added unit test for set

### PR DESCRIPTION
**Pull request by Raene**
to check if new references are created
from deepCopyAdded

### CODE REVIEW

**Comment by GrosSacASac** [comment]
please do not change code style indent with 4 spaces

**Comment by Raene** [comment]
> please do not change code style indent with 4 spaces

okay, i've fixed that in my settings. Sorry about that

File: tests/specification/deepCopy.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
+/* eslint-disable comma-dangle */
```
Comment (GrosSacASac) [inline tests/specification/deepCopy.js:1]: please do not not change the code style in a pull request about tests. If you want to change the styles open an issue, discuss about it and then maybe make a separate PR for it

File: tests/specification/deepCopy.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 import test from "ava";
 import { deepCopy, deepCopyAdded } from "../../deep.js";
 import { primitives } from "../helper.js";
 
-
-const runBaselineDeepCopyTests = (deepCopyImplementation) => {
+const runBaselineDeepCopyTests = deepCopyImplementation => {
```
Comment (GrosSacASac) [inline tests/specification/deepCopy.js:6]: 2 blank lines after exports/imports

**Review by GrosSacASac — CHANGES_REQUESTED** [review]

File: tests/specification/deepCopy.js (hunk 1 above)
Comment (Raene) [reply tests/specification/deepCopy.js:1]: okay. Noted. I think my default formatter changed it though.

File: tests/specification/deepCopy.js (hunk 2 above)
Comment (Raene) [reply tests/specification/deepCopy.js:6]: Fixed that in recent commit

Comment (GrosSacASac) [reply tests/specification/deepCopy.js:6]: Did you git push ?

Comment (Raene) [reply tests/specification/deepCopy.js:6]: Yeah I did.

Comment (GrosSacASac) [reply tests/specification/deepCopy.js:6]: Last commit I see is from 9 hours ago, before I made the comment
image

Comment (Raene) [reply tests/specification/deepCopy.js:6]: Wow. I'll look into it. Sorry for the mix up
